addpage.php

```diff
@@ -4,20 +4,11 @@
 $plant_type_list = listPlantTypes(getPlantTypes($db));
 
 $error_message = '';
-$types_message = '';
+$types_message = "<div class=\"types container\">" . $plant_type_list . "</div>";
```

Comment:
if you use single quotes for the string you wont have to escape the double quotes